recipe/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "azureml_mlflow" %}
+{% set version = "1.55.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
+  sha256: 048cd60c483d778b1f52fc219a32c925706dceac871d829946099609959c0eef
+
+build:
+  number: 0
+  # skipping because of missing mlflow-skinny for s390x
+  skip: True  # [s390x]
+
+requirements:
+  host:
+    # patch tools are in host section since they are necessary only during execution of build script
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - python
+    - pip
+    - wheel
+  run:
+    - python
+    - six
+    - cryptography
+    - mlflow-skinny
+    - azure-identity
+    - azure-common
```

I can see that `six` is needed indeed,

```suggestion
    - azure-common >=1.1,<2.0.0
```

according to 

```
Requires-Dist: jsonpickle
Requires-Dist: mlflow-skinny
Requires-Dist: azure-identity
Requires-Dist: msrest >=0.6.18
Requires-Dist: azure-core !=1.22.0,<2.0.0,>=1.8.0
Requires-Dist: azure-mgmt-core <2.0.0,>=1.2.0
Requires-Dist: azure-storage-blob <=12.19.0,>=12.5.0
Requires-Dist: azure-common <2.0.0,>=1.1
Requires-Dist: cryptography
Requires-Dist: python-dateutil <3.0.0,>=2.7.3
```
in https://inspector.pypi.io/project/azureml-mlflow/1.55.0/packages/6a/f8/8f405b06492672fbe30d6d16dc46ce59c0edc63b58037cf69dcfc6290b26/azureml_mlflow-1.55.0-py3-none-any.whl/azureml_mlflow-1.55.0.dist-info/METADATA

---

Thank you!